wdl-format/src/format.rs

```diff
@@ -0,0 +1,615 @@
+//! A module for formatting WDL code.
+
+use std::fmt::Write;
+
+use anyhow::Result;
+use format_state::SPACE;
+use wdl_ast::v1::Decl;
```

Comment:
To me, it makes sense to make a `v1` module and put all of the `v1` related stuff there. Just my two cents on how it might be broken up to make it a bit more manageable.

Reply:
(this is just a suggestion, but I'm also okay with you not doing it if you don't want to)